content/en/docs/concepts/storage/volumes.md

```diff
@@ -1302,16 +1302,15 @@ persistent volume:
 
 #### CSI raw block volume support
 
-{{< feature-state for_k8s_version="v1.14" state="beta" >}}
+{{< feature-state for_k8s_version="v1.18" state="stable" >}}
 
 Starting with version 1.11, CSI introduced support for raw block volumes, which
 relies on the raw block volume feature that was introduced in a previous version of
 Kubernetes.  This feature will make it possible for vendors with external CSI drivers to
 implement raw block volumes support in Kubernetes workloads.
```

Comment:
Remove (or revise) this text to avoid framing it as a future feature.

Reply:
Rephrased to

> Vendors with external CSI drivers can implement raw block volumes support
in Kubernetes workloads.

> You can setup your PV/PVC with raw block volume support
as usual, without any CSI specific changes.